certchain/certchainutil/certchainutil.go

```diff
@@ -77,6 +80,24 @@ func ReadPrivateKeyFile(filename string) (crypto.PrivateKey, error) {
 	return signedexchange.ParsePrivateKey(pem)
 }
 
+// ReadCertificateRequestFile reads a PEM file and returns a Certificate
+// Request usable for requesting Signed Exchange certificates.
```

Comment:
GoDoc should be just "... returns a Certificate Request." This function checks nothing of SXG compatibility.

Reply:
Done.